axelrod/player.py

```diff
@@ -241,7 +241,8 @@ def clone(self):
         return new_player
 
     def reset(self):
-        """Resets history.
+        """
+        Resets history.
         When creating strategies that create new attributes then this method
         should be re-written (in the inherited class) and should not only reset
         history but also rest all other attributes.
```

Comment:
Let's sort this out separately. I'll merge this PR and put another one in for the docstring